string fleet_name
string robot_name
uint64 map_number